Lecture4/src/main/java/App.java

```diff
@@ -0,0 +1,27 @@
+public class App {
+	public static void main(String[] args) {
+		Operator newOperator = Operator.RUN;
+		while (!newOperator.equals(Operator.QUIT)) {
+			System.out.println("Enter simple expression in 'number-operator-number' format, e.g. '2 plus 2'.");
+			System.out.println("=======Operator=======");
+			System.out.println("> Type 'plus' to Add");
+			System.out.println("> Type 'minus' to Subtract");
+			System.out.println("> Type 'multiply' to Multiply");
+			System.out.println("> Type 'divide' to Divide");
+			System.out.println("> Type 'quit' to Exit");
+			System.out.println("======================");
+			Validator validator = new Validator();
```

Comment:
`Validator validator = new Validator();` - to static final field 
you shouldn't create  new instance each iteration

Reply:
fixed